### ISSUE TITLE
Use mpirun to run benchmark.py get error

### ISSUE THREAD

**Issue by yangkunx** [intel/xFasterTransformer#153]
the error detail:
run cmd:
mpirun         -n 1 numactl -N 0  -m 0 python3 benchmark.py     --token_path /data/baichuan2-13b    --model_path /data/baichuan2-13b-xft/     --prompt_path ./prompt.json     --model_name baichuan2-13b     --dtype bf16     --token_in 1024  --token_out 512     --beam_width 1     --batch_size 1     --iteration 10     --warmup 1  --padding True

error
Failed to load xft_comm_helper library from path error code: libxft_comm_helper.so: cannot open shared object file: No such file or directory

**Comment by marvin-Yu** [comment]
```shell
$ LD_LIBRARY_PATH=`pwd`/../build/:$LD_LIBRARY_PATH mpirun -n 1 numactl -N 0 -m 0 python3 benchmark.py --token_path /data/models/chatglm2-6b-hf --model_path /data/models/chatglm2-6b-cpu --prompt_path `pwd`/prompt.json --model_name chatglm2-6b --dtype bf16 --token_in 1024 --token_out 512 --beam_width 1 --batch_size 1 --iteration 10 --warmup 1 --padding True
```
setting `LD_LIBRARY_PATH` can bypass this issue, @Duyi-Wang will fix it in the feature.

**Comment by pigstars7** [comment]
I'm having the same issue and setting LD_LIBRARY_PATH can not bypass this issue for me.

**Comment by Duyi-Wang** [comment]
> I'm having the same issue and setting LD_LIBRARY_PATH can not bypass this issue for me.

```
LD_LIBRARY_PATH={path to python}/python3.8/site-packages/xfastertransformer/libxft_comm_helper.so
```
`libxft_comm_helper.so` is packed in wheel, but it fails to found this so when mpirun.

**Comment by Duyi-Wang** [comment]
This issue has been fixed in Python wheel release 1.3.1.
